python/ray/tests/test_actor_failures.py

```diff
@@ -31,6 +31,7 @@ def ray_init_with_task_retry_delay():
         "object_store_memory": 150 * 1024 * 1024,
     }],
     indirect=True)
+@pytest.mark.skipif(sys.platform == "win32", reason="Segfaults on CI")
 def test_actor_spilled(ray_start_regular):
     object_store_memory = 150 * 1024 * 1024
```

Comment:
The segfault most probably be caused by this. I have seen this happening locally with other tests which consume high memory. For example, in [this PR](https://github.com/ray-project/ray/pull/21525/files), reducing memory helps in passing the test without issues.

Reply:
We should fix that in the framework of #20376, not here, right?

Reply:
N.B. - On my machine there was no segfault though. However, if it's coming up in CI or randomly then yes, we can include the change in the PR for #20376